<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddOKEI</class>
 <widget class="QDialog" name="AddOKEI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ОКЕИ</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_Code">
      <property name="text">
       <string>Код:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_Name">
      <property name="text">
       <string>Наименование:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_YE">
      <property name="text">
       <string>Условное обозначение:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_Code">
      <property name="maxLength">
       <number>9</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_Name">
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="line_YE">
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="push_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_Ok">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
